doc/UserGuide.md

```diff
@@ -66,6 +67,32 @@ Examples:
 * `find Betsy Tim John`<br>
   Returns Any person having names `Betsy`, `Tim`, or `John`
 
+### Editing a person : `edit`
+Edits the contact information of the specified person from the address book. Irreversible.<br>
+Format: `edit INDEX NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...`
+
+> Using it's last displayed index from the address book, to update the information of the identified person
+> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional,
+> items with `...` after them can have multiple instances. Order of parameters are fixed.
+>
+> Put a `p` before the phone / email / address prefixes to mark it as `private`. `private` details can only
+> be seen using the `viewall` command.
+>
+> Persons can have any number of tags (including 0)
+> Persons will sorted with name in alphabetical order after being added
+
+Examples:
+* `edit 1 John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
+* `edit 2 Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
+
+Examples:
```

Comment:
Is this set of examples related to edit command?

Reply:
yes